modules/monitoring-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-limits-and-resource-requests-for-monitoring-components_{context}"]
+= Specifying limits and requests for monitoring components 
+
+To configure CPU and memory resources, specify values for resource limits and requests in the appropriate `ConfigMap` object for the namespace in which the monitoring component is located:
+
+* The `cluster-monitoring-config` config map in the `openshift-monitoring` namespace for core platform monitoring.
```

Comment:
```suggestion
* The `cluster-monitoring-config` config map in the `openshift-monitoring` namespace for core platform monitoring
```
I don't think this fragment needs ending punctuation

Reply:
Removed